### DISCUSSION TITLE
Gallery view (single link for multiple files)

### DISCUSSION THREAD

**Discussion by Ivan-Malinovski** [diced/zipline#290] [Ideas]
First of, this software is amazing and I already love it after messing with for ten minutes.

But what would make it even better is the ability to upload several files, and see them on a single page. Similar to what happens on Imgur ([example](https://imgur.com/a/HdgsnDJ)), when you upload multiple images at once.

In my opinion, it'd make for a nicer user experience to have the option to share one link instead of three links.

In the case of files, it could also give the recipient a chance to see the file size (and other details) before downloading.

**Comment by diced** [comment]
Good idea, I have already added the functionality to create folders within the latest version, all that needs to be done is a way to make them public and viewable to everyone.
